modules/ossm-operatorhub-remove.adoc

```diff
@@ -137,6 +137,7 @@ Replace `<operator-project>` with the name of the project where the {ProductName
 +
 ----
 $ oc delete validatingwebhookconfiguration/<operator-project>.servicemesh-resources.maistra.io
+oc delete mutatingwebhoookconfigurations/<operator-project>.servicemesh-resources.maistra.io
```

Comment:
@gbaufake Here is the added delete command. I think I just need your review here. If you like you can review this whole PR, but if you don't have time, this is what I tagged you for.